Exiled.CustomItems/API/Features/CustomWeapon.cs

```diff
@@ -301,6 +306,12 @@ private void OnInternalHurting(HurtingEventArgs ev)
                 return;
             }
 
+            if (Server.FriendlyFire && !AllowFF && ev.Attacker.Role.Team == ev.Target.Role.Team)
```

Comment:
The Server.FriendlyFire check here is redundant. If that is false, this event will not be called.